api/src/test/java/org/motechproject/nms/api/ut/MobileAcademyControllerUnitTest.java

```diff
@@ -71,7 +73,7 @@ public void testBookmarkNullCallId() {
     public void testBookmarkMinCallId() {
         SaveBookmarkRequest sb = new SaveBookmarkRequest();
         sb.setCallingNumber(BaseController.SMALLEST_10_DIGIT_NUMBER);
-        sb.setCallId(99999999999999L);
+        sb.setCallId(VALID_CALL_ID.substring(1));
```

Comment:
terrible test name to blame, it should throw an exception because the call id is not valid. Will update